### PR TITLE
[LLM]Reopen autotp generate_stream

### PR DESCRIPTION
**Pull request by Romanticoseu**
## Description

Add `generate_stream()` API for `DeepSpeed AutoTp FastApi` example

### 2. User API changes

Implement a class called `TextIteratorStreamer`, adapted from transformers `TextIteratorStreamer`.

### 4. How to test?
- [x] Application test

### CODE REVIEW

**Review by xiangyuT — APPROVED** [review]
LGTM

File: python/llm/example/GPU/Deepspeed-AutoTP-FastAPI/run_llama2_7b_chat_hf_arc_2_card.sh [hunk 1]
```diff
@@ -33,5 +33,4 @@ export TORCH_LLM_ALLREDUCE=0
 export WORLD_SIZE=2
```
Comment (glorysdj) [inline python/llm/example/GPU/Deepspeed-AutoTP-FastAPI/run_llama2_7b_chat_hf_arc_2_card.sh:33]: rename to start-deepspeed-autotp-ipex-llm-serving.sh?

Comment (Romanticoseu) [reply python/llm/example/GPU/Deepspeed-AutoTP-FastAPI/run_llama2_7b_chat_hf_arc_2_card.sh:33]: Resolved

**Review by glorysdj — APPROVED** [review]
LGTM

**Pull request merged**